pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "autopkg"
+authors = []
+version = "2.7.3"
```

Comment:
can this be sourced from another file? I know it is technically possible, but would be nice if the version was single sourced.

Reply:
Version is in plist which is incompatible with setuptools

Reply:
where is the plist that contains the version? I was looking for it.